views/partials/banner_breach.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="account">
+  <div id="breach-title-wrapper">
+    <div class="image-wrap">
+      <img alt="placeholder"src="img/{{ breach.name }}.png">
```

Comment:
nit: Should we use `alt="{{ breach.name }}"` instead of `alt="placeholder"`? (or even an empty `alt=""` since the logo alt doesn't add much in the way of a11y in this case.